### DISCUSSION TITLE
Paid Support

### DISCUSSION THREAD

**Discussion by mrwhy-orig** [apache/plc4x#1445] [General]
Hi plc4x Team,

do you have somekind of emergency team in place for paid support inquiries? 
For instance if I'd like to speed up a bug fix and pay for it? Don't get me wrong, I'm already diving into the code, but I'm somekind of new to this codebase. 

Thanks!

**Comment by ottlukas** [comment]
You can checkout this: https://plc4x.apache.org/users/commercial-support.html

**Reply by mrwhy-orig** [reply]
Ahh thanks! Didn't see it.

**Comment by glcj** [comment]
Hello,

For the Siemens S7 driver, we can talk.

Kind regards,

**Reply by mrwhy-orig** [reply]
Great! Would you provide me a contact address?

**Reply by glcj** [reply]
Hello,

My email: cesar.garcia@ceos.com.ve

Kind regards,
